### DISCUSSION TITLE
How to suppress headers from the `toCSV()` output?

### DISCUSSION THREAD

**Discussion by agladysh** [uwdata/arquero#367] [Q&A] [answered]
I am working with CSV data with Arquero.

My input data does not have headers. But `toCSV()` output includes them. Is there a way to tell `toCSV()` to omit the header line?

```csv
armut
erik
çilek
üzüm
şeftali
```

```js
aq.fromCSV(csv, { header: false }).toCSV()
```

```csv
col1
armut
erik
çilek
üzüm
şeftali
```

**Comment by jheer** [comment]
There is not currently an option for this. I'll open a feature request issue. In the meantime, you could strip the first line, but if and only if you don't have any multi-line text strings in your data.

**Comment by jheer** [comment, marked as answer]
[Arquero version 7.1.0](https://github.com/uwdata/arquero/releases/tag/v7.1.0) now includes a `header` option for `toCSV`.

**Reply by agladysh** [reply]
Wow! Many thanks for the prompt support! Much appreciated!
